scrape/scrape.go

```diff
@@ -147,6 +149,16 @@ func newScrapePool(cfg *config.ScrapeConfig, app storage.Appendable, offsetSeed
 		return nil, fmt.Errorf("error creating HTTP client: %w", err)
 	}
 
+	validationScheme, err := config.ToValidationScheme(cfg.MetricNameValidationScheme)
```

Comment:
if we moved the blank string handling here, then we would avoid having to set the schemes every time we create an options struct

Reply:
thanks!